imports/plugins/included/shipping-shippo/register.js

```diff
@@ -31,6 +35,12 @@ Reaction.registerPackage({
     provides: ["shippingSettings"],
     container: "connection",
     template: "shippoSettings"
+  }, {
+    label: "Shippo",
+    icon: "fa fa-plane",
+    name: "shipping/addressValidation/shippo",
+    provides: ["addressValidation"],
+    template: "shippoSettings"
```

Comment:
@jshimko Will this need a migration?

Reply:
@spencern Since everyone is out can you help here?

Reply:
@Akarshit what do you need help with?

Reply:
@spencern I am not familiar with how change in registering a template works. So I have added a couple of fields above. Will they be automatically read or will they need a migration?

Reply:
@spencern could you have a look at this.

Reply:
I don't think that updates to the registry require a migration, but it might be required to update the template being requested